versioned_docs/version-2.7/integrations-in-rancher/cloud-marketplace/cloud-marketplace-payg-integration.md

```diff
@@ -0,0 +1,261 @@
+2. [Install the Rancher Prime PAYG offering on the Azure Marketplace](../cloud-marketplace/azure-marketplace-payg-integration/installing-rancher-prime.md).
+
+## FAQ
+
+The following is a list of frequently asked questions.
+
+### Marketplace Listing
+
+#### What is the Rancher Prime listing on the cloud marketplace?
+
+By selecting the Rancher Prime listing from the AWS or Azure Marketplace, you can deploy Rancher to your Kubernetes environment with the advantage of having monthly billing via the cloud.
+
+#### Where do I find the Rancher Prime listings?
+
+There are two listings in each respective cloud marketplace. They are:
+
+**AWS**
+- [Rancher Prime](https://aws.amazon.com/marketplace/pp/prodview-f2bvszurj2p2c)
+- [Rancher Prime (EMEA Orders Only)](https://aws.amazon.com/marketplace/pp/prodview-ocgjwd5c2aj5i)
+
+**Azure**
+- [Rancher Prime with 24x7 Support](https://portal.azure.com/#view/Microsoft_Azure_Marketplace/GalleryItemDetailsBladeNopdl/id/suse.rancher-prime-llc/selectionMode~/false/resourceGroupId//resourceGroupLocation//dontDiscardJourney~/false/selectedMenuId/home/launchingContext~/%7B%22galleryItemId%22%3A%22suse.rancher-prime-llcpay-as-you-go%22%2C%22source%22%3A%5B%22GalleryFeaturedMenuItemPart%22%2C%22VirtualizedTileDetails%22%5D%2C%22menuItemId%22%3A%22home%22%2C%22subMenuItemId%22%3A%22Search%20results%22%2C%22telemetryId%22%3A%221dafcf16-920e-46ea-80c9-dc85c6bd3a17%22%7D/searchTelemetryId/c2300fb7-ba7b-462a-ba57-a37cb5e2822d)
+- [Rancher Prime with 24x7 Support (EMEA Orders Only)](https://portal.azure.com/#view/Microsoft_Azure_Marketplace/GalleryItemDetailsBladeNopdl/id/suseirelandltd1692213356027.rancher-prime-ltd/selectionMode~/false/resourceGroupId//resourceGroupLocation//dontDiscardJourney~/false/selectedMenuId/home/launchingContext~/%7B%22galleryItemId%22%3A%22suseirelandltd1692213356027.rancher-prime-ltdpay-as-you-go%22%2C%22source%22%3A%5B%22GalleryFeaturedMenuItemPart%22%2C%22VirtualizedTileDetails%22%5D%2C%22menuItemId%22%3A%22home%22%2C%22subMenuItemId%22%3A%22Search%20results%22%2C%22telemetryId%22%3A%22c6b1d79a-b577-47b0-90e5-41e6c49688ab%22%7D/searchTelemetryId/1793144d-e0d9-466e-8e36-dfeddc73163b)
+
+#### Why are there two listings? Which one should I use?
+
+We have two listings for Rancher Prime, a EU/EMEA version and a non-EU, non-UK version, due to different regulations in different locations. Whichever listing you choose, the images and support offered are identical. You should pick the listing for the location of your cloud provider account.
+
+#### Are these listings available in all countries?
+
+No. The PAYG listing isn't available in all countries.
+
+Your billing country is based on the cloud provider Account ID for the deployment.
+
+Please refer to [Geographical availability](https://documentation.suse.com/sle-public-cloud/all/html/public-cloud/countrylist.html#) for a list of countries that can and cannot access Rancher Prime via the AWS and Azure cloud marketplaces.
```

Comment:
The `can transact` list in this link differs from the list being removed from `cloud-marketplace/aws-marketplace-payg-integration/aws-marketplace-payg-integration.md`. The new link lists `Viet Nam (VN): non-EU and non-UK` whereas that was not part of the explicit list being removed. Is this link the source for the original list that was in the AWS FAQ?

Reply:
Stephen asked to use https://documentation.suse.com/sle-public-cloud/all/html/public-cloud/countrylist.html as a single source of truth; it has been and will be updated moving forward. @stephenmogg may you confirm this is the case? Thanks!